docs/asciidoc/coreFeatures.adoc

```diff
@@ -47,7 +47,11 @@ The following image shows how a `Mono` transforms an item:
 image::https://raw.githubusercontent.com/reactor/reactor-core/v3.0.7.RELEASE/src/docs/marble/mono.png[Mono]
 
 A `Mono<T>` is a specialized `Publisher<T>` that emits at most one item and then
-(optionally) terminates with an `onComplete` signal or an `onError` signal.
+terminates with an `onComplete` signal or alternatively, emits an `onError` signal.
+Most `Mono` implementations are expected to immediately call `onComplete` on their
+`Subscriber` after having called `onNext`. `Mono.never()` is an outlier: it doesn't
+emit any signal, which is not technically forbidden although not terribly useful outside
+of tests. On the other hand, `onNext+onError` is explicitly forbidden.
```

Comment:
```suggestion
of tests. On the other hand, `onNext` + `onError` is explicitly forbidden.
```

Reply:
Tried that, but this leaves the room for a line change that I wanted to avoid. Your call.

Reply:
ah I see, ugly line breaks is the issue. let me suggest a third phrasing option then:

```suggestion
of tests. On the other hand, a combination of `onNext` and `onError` is explicitly forbidden.
```